### ISSUE TITLE
Australia Server is fucked

### ISSUE THREAD

**Issue by Chadzxc13** [ValveSoftware/Dota2-Gameplay#2158]
### Description

Getting disconnected every 2-3 games. Please fix this it pisses me off and it always happen when I'm winning. 

### Example Match ID (and possibly Timestamp)

6715299434

**Comment by Akz77** [comment]
Also match ID 6715562872
watch from 32:50 seconds (near the end)

**Comment by bigdaddydee89** [comment]
Yup. Just had my 12th game do it. Last two within the same hour. Are we gonna get a answer?

**Comment by commentator24** [comment]
#2151 #2152

**Comment by sgtslick** [comment]
AUS server: Happening a lot for me too, match id: 6715563563, 6715859769, 6714663275, 6714033305
